ktor-io/common/src/io/ktor/utils/io/core/AbstractInput.kt

```diff
@@ -314,13 +314,13 @@ abstract class AbstractInput(
     }
 
     @Deprecated("Binary compatibility.", level = DeprecationLevel.HIDDEN)
-    final override fun readShort() = readShort()
+    final override fun readShort(): Short = readShort()
```

Comment:
Should `public` modifier be here?

Reply:
Nope. We should set visibility modifiers for `overrides` only when it changes visibility